spec/os_spec.rb

```diff
@@ -121,7 +121,8 @@
   it "has working cpu count method" do
     cpu_count = OS.cpu_count
     assert cpu_count >= 1
-    assert (cpu_count & (cpu_count - 1)) == 0 # CPU count is normally a power of 2
```

Comment:
I use a 6-core, 12 vCPU Macbook Pro.  The power of 2 idea is no longer reasonable.